src/Commands/Configure.ts

```diff
@@ -46,15 +36,20 @@ export class Configure implements CommandModule {
       })
       .middleware((args: Arguments) => {
         container.register(ConnectivityUrls, {
-          useValue: new Map(
-            Object.values(TestType).map((type: TestType) => {
-              try {
-                return [type, new URL(args[type] as string)];
-              } catch (err) {
-                throw new Error(`Invalid value for ${type} testing endpoint`);
-              }
-            })
-          )
+          useValue: new Map([
+            this.getMapEntryOrThrow(
```

Comment:
It seems you will get an exception. You can't use instance methods in commands excluding build-in